node/test/toolrunnertests.ts

```diff
@@ -35,6 +35,7 @@ describe('Toolrunner Tests', function () {
             cwd: __dirname,
             env: {},
             silent: false,
+            hideCommand: false,
```

Comment:
NIT - If the parameter's default value is false, its declaration may be omitted.

Reply:
Updated